src/interfaces/IMassaProtoFile.ts

```diff
@@ -5,7 +5,8 @@
  * @see candidate_value - If it is different from the final_value, it represents the
  * candidate file (as a byte array) to replace the current proto file.
  */
-export interface IProtoFile {
-  final_value: Uint8Array;
-  candidate_value: Uint8Array;
+export interface IMassaProtoFile {
```

Comment:
Can you remove the useless I prefix for the interface?
If there is more, can you create an issue about that ?